backend/prisma/migrations/20201220171122_init/migration.sql

```diff
@@ -14,15 +17,13 @@ CREATE TABLE "User" (
     "firstname" TEXT,
     "lastname" TEXT,
```

Comment:
I noticed all the good apps are using a single field for name. Like Figma, etc. 

Then I remembered that asking for first name and last name has other issues as well: 
- In some countries the first name is actually the last name, like in China
- When searching by full names we'll need joining them anyway 

I suggest we go by a single field like Figma: 
Name

Reply:
Fixed in #278, I will rebase this and drop related changes soon.